upi/azure/04_bootstrap.json

```diff
@@ -19,7 +19,7 @@
     "sshKeyData" : {
       "type" : "securestring",
       "metadata" : {
-        "description" : "SSH RSA public key file as a string."
+        "description" : "Unused"
```

Comment:
These should have a default value specified as well so that users do not have to supply a dummy value as so that we can eventually remove the parameter once support for older versions make their way out of CI.